### ISSUE TITLE
label phone numbers as work/home/etc. (feature)

### ISSUE THREAD

**Issue by bbrendon** [monicahq/monica#3569]
Currently you can enter many phone numbers for a contact in monica. You can not label them as mobile/work/etc. 

Also, would be nice if the labels synced over CardDAV to iOS/Mac/Whatever.

**Comment by pqhf5kd** [comment]
<img width="828" alt="image" src="https://user-images.githubusercontent.com/20027419/74586222-c5a12980-4fe5-11ea-8d8e-b0e216871ee6.png">

You can add additional labels in Settings as above but they don't sync over CardDAV.

**Comment by bbrendon** [comment]
Ahh... but that seems only kinda helpful. When you view a contact, it doesn't have text describing what it is and it seems you can't change the icon. When I try `fa fa-phone-office` or `fas fa-mobile-alt`, I get no icon.

**Comment by pqhf5kd** [comment]
> Ahh... but that seems only kinda helpful. When you view a contact, it doesn't have text describing what it is and it seems you can't change the icon. When I try `fa fa-phone-office` or `fas fa-mobile-alt`, I get no icon.

That was fixed here, you’re hosting your own instance correct? 
https://github.com/monicahq/monica/pull/3194

> 
> Also, like you said on my mobile I have no idea which is which.

Agreed

**Comment by bbrendon** [comment]
I'm running 2.16.0 which I'm guessing that is included in but I don't know how to find out.

**Comment by bbrendon** [comment]
Yes. I used a FA v4 icon and it works!! How do you tell it to use non FA Icons? 

I read through github and it says these work : https://thenounproject.com/
I'm not sure how to specify non FA icons.

Search for FAv4 here: https://fontawesome.com/v4.7.0/icons/

**Comment by bbrendon** [comment]
But.... now phone numbers are disappearing from my phone. I guess if you have multiple types of `tel:` then only one will sync. Looks like I'm going to have to revert back to one unlabeled phone number.

**Comment by LunkSnee** [comment]
Definitely need to be able to include home/mobile/work phone numbers and emails, and sync over CardDav. I really can't use Monica well until then.

**Comment by asbiin** [comment]
Some work has started to do that. It will be finished soon.
Track this issue: #3894
